K23003

akki-256 opened the pull request:
アプリ本体の追加
gitignoreの追加
READMEに記入

akki-256 committed:
add:初期コメット’

akki-256 committed:
update:タイポ修正，README改定

akki-256 committed:
update:重複箇所の削除

akki-256 committed:
update
resgiter->resister
cont->ctrl

akki-256 committed:
update:不足要素の追加

kazyam53 commented on db/requirements.txt:
```diff
@@ -0,0 +1,5 @@
+fastapi
+websockets
+contextlib
+sqlite3
```
これはWindowsで試されたときのオプションでしょうか？
ubuntuで試した際に、
contextlib→contextlib2
sqlite3→install不可
でした

kazyam53 reviewed (changes requested):
テスト用に、サンプルのバーコードの画像と、それをdbに登録するための情報(もしくはサンプルのdbでの起動オプション)を追加してもらえますでしょうか。

akki-256 committed:
updates
installするライブラリの見直し
テスト用バーコードの追加(./db/sampleBarcodes/)
サンプル用のデータベースの追加

akki-256 committed:
update:サンプルバーコード設定ミス

akki-256 reviewed (commented):
対応しました
sqlite3とcontextlibはpythonに標準ライブラリだったためにエラーを起こしていたと思われます
サンプル用バーコードは./db/sampleBarcodes/の配下に３種類追加しました
また，サンプル用バーコードをデータベースに追加(データベースがない場合は新規作成)するコードを作成しました．db.pyを実行した後，出力の指示に従い'sample'と入力するとデータベースにサンプルバーコードが書き込まれます

akki-256 commented:
誤操作です

kazyam53 commented on AKARI/requirements.txt:
```diff
@@ -0,0 +1,4 @@
+opencv-python
+pyzbar
+requests
+akari-client
```
```suggestion
akari-client[depthai]

```
depthai, blobconverterなどのライブラリが不足なので、上記を追加して下さい。

kazyam53 reviewed (changes requested):
AKARI/main.pyを起動するとlib/ディレクトリがないと表示されます。
またconfig/barcode.jsonやmodel/barcode.jsonも含まれていないので起動できません。

akki-256 committed:
Update requirements.txt

akki-256 committed:
Update README.md

redisコンテナ起動時にroot権限で開始するように変更

akki-256 committed:
Update README.md

記述漏れのある操作を追加

kazyam53 commented:
@akki-256 
上記で依頼した点が直っていないように思えます…。

> AKARI/main.pyを起動するとlib/ディレクトリがないと表示されます。
> またconfig/barcode.jsonやmodel/barcode.jsonも含まれていないので起動できません。

akki-256 committed:
update
util.py,barcode.json,.blob,oakd_yolo.pyの追加
requirements.txtにtqdmの追加
README.mdのredis起動に関する追記

akki-256 reviewed (commented):
お待たせしました
- lib/,congfig/...,model/...を追加
- pip install項目にtqdmの追加
- README.mdに主にredisの起動に関する追記

を行いました

kazyam53 reviewed (approved):
OKです。ありがとうございます！
